dlc-oracle/src/main/scala/org/bitcoins/dlc/oracle/DLCOracle.scala

```diff
@@ -372,6 +372,25 @@ class DLCOracle(private[this] val extPrivateKey: ExtPrivateKeyHardened)(implicit
       digitSigs <- Future.sequence(digitSigFs)
     } yield OracleEvent.fromEventDbs(signSig ++ digitSigs)
   }
+
+  override def deleteSigs(eventName: String): Future[OracleEvent] = {
```

Comment:
I would add those same scaladocs here just to be SUPER clear that this is a possibly dangerous method